MTALogWatcher.py

```diff
@@ -0,0 +1,579 @@
+#!/usr/bin/env python
+from __future__ import division
+import argparse
+import os
+import re
+import sys
+import datetime
+import json
+
+import DefsParser
+from Config import Config
+from Errors import *
+from Utils import mkdirp
+
+#TODO: implement a status to match "log-only = true" results
+MANDATORY_STATUS_TAGS = [ "sent_ok", "sent_ko", "attempted" ]
+
+class MTALogWatcher():
```

Comment:
@azet we talked about working on one of these.  I think this is a reasonable thing to start a conversation around -- if we want to help people parse the logs and review them for bounces like I mentioned I was seeing from prominent ISP mail domains like comcast and sbcglobal.

If we want something that the user can simply run this could work -- and we clean up the date regexs maybe use dateutil, and make a couple of small classes to simplify holding the parsed data.

On the other hand we could make a site on top of simple http server to show the data in a slightly more friendly way.  Though I think that would still leave us working on some parsing logic like we see below, and then stuff it into json, something a frontend can use.

From that we could allow the user to easily configure upstream reporting of domains failing basic TLS handshakes to an EFF server, or something.  And then send the mail admins of the domains an email once a month or with some stats on affected domains/users/delivery and materials on better TLS configurations.